benchmarks/nemo/tracer_advection/original/omp_trans.py

```diff
@@ -0,0 +1,73 @@
+# -----------------------------------------------------------------------------
```

Comment:
Why is this file not in the scripts directory?

Reply:
A good question - I've moved it.